launch/phone_ui.launch

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<launch>
+    <arg name="namespace" default="/hero" />
+
+    <group ns="$(arg namespace)">
```

Comment:
This would imply the phone ui is not launched on the real robot

Reply:
No, on the robot it was launched in a separate launch file, probably in a service. So it was always running, before `hero-start`.

Reply:
A separate launch file was created for the phone ui. But I believe this was done for testing purposes. There is no service which launches the ui.

Reply:
This one is just launching the server, no actual UI. I will check on the robot tonight if any service is launching this.

Reply:
I added the service which launches this to scripts.